data/json/mutations.json

```diff
@@ -3925,7 +3925,11 @@
         "description" : "Your body's ability to digest fruits, vegetables and grains is severely hampered.  You cannot eat anything besides meat.",
         "cancels" : ["VEGETARIAN", "HERBIVORE", "RUMINANT",  "GRAZER"],
         "leads_to" : ["SAPROVORE"],
-        "category" : ["MUTCAT_LIZARD", "MUTCAT_BEAST", "MUTCAT_SPIDER", "MUTCAT_CHIMERA", "MUTCAT_RAPTOR", "MUTCAT_FELINE"]
+        "category" : ["MUTCAT_LIZARD", "MUTCAT_BEAST", "MUTCAT_SPIDER", "MUTCAT_CHIMERA", "MUTCAT_RAPTOR", "MUTCAT_FELINE"],
+        "vitamin_rates": [
+            [ "VitA", -999 ],
```

Comment:
Just make vitamin A not cause a deficiency until it is properly implemented. Same for calcium, which currently only exists in animal sources. Focus on 2 vitamins that work rather than adding hacks to make the rest pretend to work.